### PR TITLE
Added NetworkforceStopInputs

### PR DESCRIPTION
**Pull request by NaoUnderscore**
I don't know how to rename it, I think it might be useful tho.

### CODE REVIEW

File: Exiled.API/Features/Player.cs [hunk 1]
```diff
@@ -908,7 +908,16 @@ public int Scp330Usages
         /// Gets player's items.
         /// </summary>
         public Inventory.SyncListItemInfo Items => Inventory.items;
-
+        
+        /// <summary>
+        /// Gets or sets a value indicating whether or not the player can send inputs.
+        /// </summary>
+        public bool NetworkforceStopInputs
```
Comment (PintTheDragon) [inline Exiled.API/Features/Player.cs:915]: This name's kinda gross. Maybe go with something more descriptive like `CanSendInputs`.

**Review by PintTheDragon — CHANGES_REQUESTED** [review]

File: Exiled.API/Features/Player.cs (hunk 1 above)
Comment (Nekonyx) [reply Exiled.API/Features/Player.cs:915]: Or `CanMove`

Comment (NaoUnderscore) [reply Exiled.API/Features/Player.cs:915]: Yeah I think you're right. `CanSendInputs` is fine.

**Review by iopietro — CHANGES_REQUESTED** [review]
Fix build errors.

**Comment by NaoUnderscore** [comment]
I’ll fix it today.

**Review by BuildBoy12 — APPROVED** [review]

**Review by iopietro — APPROVED** [review]

**Pull request merged**